src/styles/utils.scss/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$card-width: 200px;
+$catalog-gap: 48px;
+
```

Comment:
do u need it as variable? in your code u use it only once.

Reply:
Valid! But I want to keep it as a variable for now. I'm not sure if it's considered bad practice, but I enjoy having them all in one place, hypothetically the value may change in the future, so it's easier for me this way.